### ISSUE TITLE
NullPointerException on close active task

### ISSUE THREAD

**Issue by Dawen18** [OpherV/gitflow4idea#197]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] puppy => You're not submitting a puppy. I already have one and he's [adorable](https://twitter.com/Opherv/status/974341249099542528)

* **What is the current behavior?**
NullPointerException on close task with or without creation of feature.

* **Is this a bug? Sorry about that. If so give me explicit details how to reproduce:**

1. Open IntelliJ
2. Start task with or without "Create feature" checked in Gitflow section
3. Close active task (with shortcut or Tool -> Tasks & Context -> Close active Task)
4. The panel not open and Exception is thrown (Stack trace is below)

* **What is the expected behavior?**
Close active task for open new.

* **What is the motivation / use case for changing the behavior?**

* **Please tell me about your environment:**
  
  - Gitflow4idea version: 0.6.5.1
  - Gitflow version: 0.4.1
  
  - *IntelliJ Help -> about > click copy icon and paste here. Should look like this:*
  ~~~~
    PhpStorm 2018.1
    Build #PS-181.4203.565, built on March 28, 2018
    Licensed to Consulting Convergence & System SA / David Wittwer
    Subscription is active until May 7, 2018
    JRE: 1.8.0_152-release-1136-b20 x86_64
    JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
    macOS 10.13.3
  ~~~~

```
java.lang.NullPointerException
	at gitflow.ui.GitflowCloseTaskPanel.<init>(GitflowCloseTaskPanel.java:64)
	at gitflow.ui.GitflowTaskDialogPanelProvider.getCloseTaskPanel(GitflowTaskDialogPanelProvider.java:41)
	at com.intellij.tasks.ui.TaskDialogPanelProvider.lambda$getCloseTaskPanels$1(TaskDialogPanelProvider.java:44)
	at com.intellij.util.containers.ContainerUtil.mapNotNull(ContainerUtil.java:1925)
	at com.intellij.util.containers.ContainerUtil.mapNotNull(ContainerUtil.java:1874)
	at com.intellij.tasks.ui.TaskDialogPanelProvider.getCloseTaskPanels(TaskDialogPanelProvider.java:43)
	at com.intellij.tasks.actions.CloseTaskDialog.<init>(CloseTaskDialog.java:92)
	at com.intellij.tasks.actions.CloseTaskAction.actionPerformed(CloseTaskAction.java:39)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:576)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.a(IdeKeyEventDispatcher.java:625)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:195)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:624)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.d(IdeKeyEventDispatcher.java:479)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:213)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:738)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:434)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by Dawen18** [comment]
Hi, just for more precision.
If I finish feature with Gitflow menu on bottom right, everything works fine.
The exception append only on closing task.
Thanks

**Comment by jfeid** [comment]
Confirmed this bug also. 

PhpStorm 2018.1.1
Build #PS-181.4445.72, built on April 9, 2018
Licensed to Ioannis Feidaros
Subscription is active until November 6, 2018
JRE: 1.8.0_152-release-1136-b27 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.13.0-38-generic

**Comment by scottlimmer** [comment]
Confirmed.

PhpStorm 2018.1.3
Build #PS-181.4892.97, built on May 9, 2018
Licensed to Scott Limmer
Subscription is active until January 7, 2019
For educational use only.
JRE: 1.8.0_152-release-1136-b38 amd64
JVM: OpenJDK 64-Bit Server VM by Sun Microsystems Inc.
Windows 10 10.0

**Comment by bad79s** [comment]
Confirmed.

Android Studio 3.2 Canary 15
Build #AI-181.4668.68.32.4773949, built on May 11, 2018
JRE: 1.8.0_152-release-1136-b02 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.5

**Comment by dasginganinja** [comment]
Confirmed.

PhpStorm 2018.1.4
Build #PS-181.5087.24, built on May 18, 2018
Licensed to Thomas Donahue
You have a perpetual fallback license for this version
Subscription is active until May 4, 2019
JRE: 1.8.0_152-release-1136-b39 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.4

**Comment by pgrupp** [comment]
Confirmed.

IntelliJ IDEA 2018.1.5 (Ultimate Edition)
Build #IU-181.5281.24, built on June 12, 2018
Licensed to Patrick Grupp
Subscription is active until October 23, 2018
JRE: 1.8.0_152-release-1136-b39 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.5

**Comment by pgrupp** [comment]
I've been investigating a bit. This seems to be happening because of an option not being set, but where to set it, I'm not sure:
In the above mentioned line this is actually null: `GitflowConfigurable.getOptionTextString(project, "HOTFIX_customHotfixCommitMessage")` but the `.replace("%name%", branchName)` is called on it anyway. So I was able to fix this probably. But ran into another issue with the GitflowState being corrupted/out of sync thus not showing the correct options, when closing the task anyway. I'll keep you posted and come up with a pull request, if I succeed.
Cheers :)
